pkg/svcat/service-catalog/service-catalogfakes/fake_svcat_client.go

```diff
@@ -2744,6 +2814,8 @@ func (fake *FakeSvcatClient) Invocations() map[string][][]interface{} {
 	defer fake.retrieveClassByIDMutex.RUnlock()
 	fake.retrieveClassByPlanMutex.RLock()
 	defer fake.retrieveClassByPlanMutex.RUnlock()
+	fake.createClassMutex.RLock()
+	defer fake.createClassMutex.RUnlock()
```

Comment:
Are these two createClassMutex's supposed to be included here?

Reply:
This isn't from my change - it looks like someone mucked with the interface and didn't regenerate the fake.